Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -21,6 +22,9 @@ final class SitterProfileViewModel: ObservableObject {
     /// The price per hour for walking charged by the pet sitter.
     @Published var pricePerHour: String = ""
 
+    /// Indicating whether an error has occurred during the network operation.
+    @Published var isErrorOccurred: Bool = false
```

Comment:
Why do we need this property now?

Reply:
This property was planned to be used as a source of truth to trigger a pop-up alert in UI.

Reply:
When someone adds the alert, he will add this property.

Reply:
Fixed